CHANGELOG.md

```diff
@@ -9,7 +9,55 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
-# Changes Since v3.1.1
+# Changes Since v3.2.0
+
+# v3.2.0 - [2019.04.11]
+
+## New features / functionalities
+  - Introduced a new basic framework for creating and managing plugins
+  - Added the ability to create containers through multi-stage builds
+  - Created the concept of a "remote" SCS endpoint and added the ability for users and admins to set through conf files 
+  - Streamlined the key container signing, verifying, and key management workflows
```

Comment:
Also probably not necessary. It's pretty vague and isn't a new feature or function.